examples/katkem/main.rs

```diff
@@ -1,15 +1,17 @@
-use hex;
-
 use std::io::Write;
 use std::io::{BufRead, BufReader};
 use std::{env, error, fmt, fs};
 
 use classic_mceliece_rust::{crypto_kem_dec, crypto_kem_enc, crypto_kem_keypair};
-use classic_mceliece_rust::{AesState, RNGState};
 use classic_mceliece_rust::{
     CRYPTO_BYTES, CRYPTO_CIPHERTEXTBYTES, CRYPTO_PRIMITIVE, CRYPTO_PUBLICKEYBYTES,
     CRYPTO_SECRETKEYBYTES,
 };
+use rand::RngCore;
+
+use crate::nist_aes_rng::AesState;
+
+mod nist_aes_rng;
```

Comment:
This entire rng is duplicated in the current PR. This can be avoided by doing

```rust
#[path = "../../src/nist_aes_rng.rs"]
mod nist_aes_rng;
```

In order to import it from the existing file in the main `src/` directory

Reply:
That is a nice trick, thanks.

Reply:
Thanks to this change, the example is no longer multiple files. So you can move `main.rs` one level up and name it back to `katkem.rs`.